src/router/routes.js

```diff
@@ -97,6 +97,13 @@ export default [
 	{ path: '/lend-by-category', component: () => import('@/pages/Lend/LendByCategoryPage') },
 	{ path: '/lend-by-category/:category', component: () => import('@/pages/Lend/LoanChannelCategoryPage') },
 	{ path: '/lend/filter', component: () => import('@/pages/Lend/Filter/LendFilterPage') },
+	{
+		path: '/lp/:dynamicRoute',
+		component: () => import('@/pages/ContentfulPage'),
+		meta: route => ({
+			contentfulPage: `lp/${route.params.dynamicRoute}`,
```

Comment:
if the `/` is ok in the queries I don't see an issue with this and think it's useful to have some namespacing built into those keys seeing as they must be unique.